src/main/java/org/prebid/server/analytics/reporter/greenbids/model/GreenbidsJacksonMapper.java

```diff
@@ -0,0 +1,30 @@
+package org.prebid.server.analytics.reporter.greenbids.model;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import org.prebid.server.json.EncodeException;
+
+import java.util.Objects;
+
+public class GreenbidsJacksonMapper {
```

Comment:
We are trying to avoid the creation of custom mappers like this.
Please remove this mapper and just add `@JsonProperty` annotation to required model fields